nsv13/code/modules/overmap/ai-skynet.dm

```diff
@@ -65,6 +65,11 @@ Adding tasks is easy! Just define a datum for it.
 
 	var/list/shared_targets = list()
 
+/datum/fleet/Destroy()
+	QDEL_LIST(all_ships) //We're taking the ships with us
+	current_system.fleets -= src
+	. = ..()
```

Comment:
This will cause `defeat()` to be ran on manual deletes since `overmap/Destroy()` -> `fleet/remove_ship()` -> `fleet/defeat()` (and also will cause it to try to qdel the fleet while it's already qdeling which causes a warning)

Reply:
I see, I wasn't aware overmap Destroy called it, my bad. It was my intention to bypass defeat here yeah

Reply:
Found the most evil solution for this good luck (I tested it and it worked though)